### ISSUE TITLE
about the comparison between hrnet and rle-hrnet

### ISSUE THREAD

**Issue by WeianMao** [Jeff-sjtu/res-loglikelihood-regression#26]
hi, i am prepare to cite you paper, but i am confuse about the table2 in your paper. there are 2 issue.
first, the simple baseline in your paper is 71.0, but in the simple baseline in its official paper is 70.4.
second, the hrnet in your paper is 74.1, but the result in official paper is 74.4.
can you provide more detail of those experiment? thank you!

**Comment by jeffffffli** [comment]
Hi @maowayne123, we re-train SimplePose and HRNet with the same settings as RLE for a fair comparison. The model is trained for 230 epochs in total. The learning rate is set to 0.001 and multiplied by 0.1 at 170th and 200th epoch. Data augmentations include horizontal flip(50%), rotations(45%), scale(25%) and etc. You can refer to the [config file](https://github.com/Jeff-sjtu/res-loglikelihood-regression/blob/master/configs/256x192_res50_regress-flow.yaml) for more details.
